docs/source/essentials/custom-scalars.mdx

```diff
@@ -119,13 +119,16 @@ This method takes a type-safe generated class from `Types`, along with its corre
 
 The `com.apollographql.apollo3:apollo-adapters` artifact provides built-in adapters for the following common custom scalar types.
 
-| Adapter | Description |
-|---------|-------------|
-| `com.apollographql.apollo3.adapter.InstantAdapter` | For `kotlinx.datetime.Instant` ISO8601 dates |
-| `com.apollographql.apollo3.adapter.LocalDateAdapter` | For `kotlinx.datetime.LocalDate` ISO8601 dates |
-| `com.apollographql.apollo3.adapter.DateAdapter` | For `java.util.Date` ISO8601 dates |
-| `com.apollographql.apollo3.adapter.LongAdapter` | For `java.lang.Long` |
```

Comment:
There is a `LongAdapter` bundled in api, but not sure if it's worth mentioning.

Reply:
It's definitely going to be used more often than these ones so I'd lean towards mentioning it.